Dockerfiles/Dockerfile.frontend

```diff
@@ -1,8 +1,15 @@
-FROM node:20-bookworm
+FROM node:20-bookworm as build
 
 ENV REACT_APP_PROXY=http://localhost:9081
 ENV REACT_APP_NOMOCK=on
 WORKDIR /web/frontend
 COPY ../web/frontend .
-RUN npm install
-ENTRYPOINT ["npm", "start"]
+RUN npm ci
+RUN npm run build
+
+FROM node:20-bookworm as app
+
+WORKDIR /web/frontend
+COPY --from=build /web/frontend/build/ .
+
+ENTRYPOINT ["python3", "-m", "http.server", "3000", "--directory", "/web/frontend/", "--bind", "127.0.0.1"]
```

Comment:
I see you removed `npm start` in favor of building the static site. There's one caveat to that, which is that during development, you will need to rebuild every time you do a change. 

With no `npm start`, you can get hot reloading, or at least you would need to do `docker compose restart container` 
With this design, you would need `docker .... build` first then `docker .... up` second.  

but... maybe,  if we change the entrypoint to a CMD, we can override it in compose file to a `npm run`

Reply:
please see @ineiti 's comments, now that we want to actually deploy it I agree that it makes sense to have a clean Docker image

personally, I have a separate Dockerfile for developing that kept the `npm start` for the reasons you mentioned

Reply:
You can also add a `Dockerfile.dev` with the `npm start` in it.

For the `npm start`, do you use the `-v` to link your local directory?